mmcv/cnn/bricks/conv_module.py

```diff
@@ -105,7 +105,8 @@ def __init__(self,
         self.with_bias = bias
 
         if self.with_norm and self.with_bias:
-            warnings.warn('ConvModule has norm and bias at the same time')
+            if self.conv_cfg.get('type') in ['BN', 'BN1d', 'BN2d', 'BN3d']:
```

Comment:
Can we change the `self.conv_cfg.get('type')` to `isinstance`

Reply:
Could you please give the exact code you want to use?  I think we can only access the type string at this point, not an instance of norm ?

Reply:
yet, I miss the type of self.conv_cfg

Reply:
However, I do have a more elegant way, haha, push finished.